Style.css

```diff
@@ -1,3 +1,23 @@
 body {
    background-color: black;
 }
+header{
+   display: flex;
+   justify-content: space-between;
+   padding:8px;
+}
+.main-nav{
+   display:flex;
+   align-items: baseline;
+}
+.nav-link{
+margin: 12px;
+}
+nav li a{
+   color:white;
+   font-style:arial;
```

Comment:
define font style css at body level, and if you want to change the style, then override it in any other css class